doc/source/developer_guide/dg_driver.rst

```diff
@@ -7,22 +7,37 @@ Driver and Coupling
 ====================
 
 The driver and coupling layer is found in **cicecore/drivers/**.  The standalone driver is found
-under **cicecore/drivers/cice/** and other high level coupling layers are found in other directories.
-In general, CICE will build with only one of these drivers, depending how the model is run and
-coupled.  Within the **cicecore/drivers/cice/** directory, the following files are found,
+under **cicecore/drivers/standalone/cice/** and other high level coupling layers are found in other directories.
+CICE is designed to build with only one of these drivers at a time, depending how the model is run and coupled.  Within the **cicecore/drivers/standalone/cice/** directory, the following files are found,
 
 **CICE.F90** is the top level program file and that calls CICE_Initialize, CICE_Run, and CICE_Finalize methods.
 **CICE_InitMod.F90** contains the CICE_Initialize method and other next level source code.
 **CICE_RunMod.F90** contains the CICE_Run method and other next level source code.
 **CICE_FinalMod.F90** contains the CICE_Finalize method and other next level source code.
 
-Other **cicecore/drivers/** directories are similarly implemented with a top level coupling layer,
-that is largely specified by an external coupled system and then some version of the **CICE_InitMod.F90**,
-**CICE_RunMod.F90**, and **CICE_FinalMod.F90** files.
+The files provide the top level sequencing for calling the standalone CICE model.
+
+Adding a New Driver
+------------------------
+
+The drivers directory contains two-levels of subdirectories.  The first layer indicates the coupling infrastructure or strategy and the second later indicates the application or coupler the driver is written for.  At the present time, the directory structures looks like::
```

Comment:
"two-levels" -> "two levels"

Reply:
Good catch, this is clearly incorrect.  I have just pushed a fix.